ReSharper.FSharp/src/FSharp/FSharp.Psi/src/Impl/Tree/FSharpTypeDeclaration.cs

```diff
@@ -47,5 +47,21 @@ ITypeInherit TryFindTypeInherit(TreeNodeEnumerable<ITypeBodyMemberDeclaration> m
         };
       }
     }
+
+    public IEnumerable<IInheritMember> TypeOrInterfaceInheritMembers
+    {
+      get
+      {
+        var members = TypeRepresentation switch
+        {
+          null => TypeMembersEnumerable,
+          IClassRepresentation classRepr => classRepr.TypeMembersEnumerable,
+          IInterfaceRepresentation interfaceRepr => interfaceRepr.TypeMembersEnumerable,
+          _ => TreeNodeEnumerable<ITypeBodyMemberDeclaration>.Empty
+        };
```

Comment:
Do we need it as a part of PSI? Are there any other features, besides the AI summarization, that need it?